src/DotNet.Testcontainers.Tests/Unit/Containers/Unix/Database/CouchbaseTestcontainerTest.cs

```diff
@@ -0,0 +1,157 @@
+namespace DotNet.Testcontainers.Tests.Unit.Containers.Unix.Database
+{
+  using System;
+  using System.Threading.Tasks;
+  using Couchbase;
+  using Testcontainers.Containers.Builders;
+  using Testcontainers.Containers.Configurations.Databases;
+  using Testcontainers.Containers.Modules.Databases;
+  using Xunit;
+
+  public class CouchbaseTestcontainerTest
+  {
+
+    /// <summary>
+    /// Sample Couchbase document class for test
+    /// </summary>
+    private class Customer
+    {
+      public string Name { get; set; }
+      public int Age{ get; set; }
+    }
+
+    [Fact]
+    public async Task ConnectionEstablished_WithCrudTest()
+    {
+      //Given
+      var couchbaseTestcontainer = new TestcontainersBuilder<CouchbaseTestcontainer>()
```

Comment:
Add `using` statement. Maybe also consider `cluster` and `bucket`. Or better use the `OracleFixture` approach with `IAsyncLifetime`.

Reply:
IAsyncLifetime looks like more better solution for manage. I'll update.